Dockerfile

```diff
@@ -1,33 +1,37 @@
-FROM alpine:3.10
-MAINTAINER Victor Trac <victor@cloudkite.io>
+FROM alpine:3.15.4
+LABEL MAINTAINER Victor Trac <victor@cloudkite.io>
 
-ENV VERSION="1.29.2547.95"
+ENV VERSION="1.30.3116.68"
+
+RUN apk -U upgrade
 
 # Build deps
-RUN apk --no-cache add --update go git bzr wget py2-pip \ 
-    gcc python python-dev make musl-dev linux-headers libffi-dev openssl-dev \
+RUN apk --no-cache add --update go git wget py3-pip \ 
+    gcc python3 python3-dev cargo make musl-dev linux-headers libffi-dev openssl-dev \
```

Comment:
I added cargo because I was getting errors:

```
Building wheel for cryptography (pyproject.toml) did not run successfully
error: can't find Rust compiler
```